src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -614,6 +614,7 @@
     <Compile Include="$(BclSourcesRoot)\mscorlib.Friends.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="shared\System\Buffers\ValueListBuilder.cs" />
```

Comment:
Could you please add this to src\mscorlib\shared\System.Private.CoreLib.Shared.projitems instead. (More details: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/README.md)